platforms/hyperledger-indy/configuration/roles/setup/add_did_vault/README.md

```diff
@@ -0,0 +1,11 @@
+## setup/endorsers
```

Comment:
setup/add_did_vault is not correct name, either use setup/did_vault or add/did_vault